File: app/lib/screens/InvoicePage/models/Payment.dart
class Payment {
  String date;
  int amount;
  String mode;
  String id;

  Payment(this.amount, this.date, this.mode, this.id);

  Payment.fromJson(Map<String, dynamic> json) {
    date = json['date'].toString().split('T')[0];
    amount = json['amount'];
    mode = json['mode'].toString();
    id = json['_id'];
  }
}

File: app-backend/models/Payment.js
const mongoose = require('mongoose')

const paymentSchema = new mongoose.Schema({
  amount: {
    type: Number,
    required: true
  },
  date: {
    type: Date,
    default: Date.now()
  },
  mode: {
    type: String,
    required: true
  }
})

module.exports = mongoose.model('Payment', paymentSchema)
